src/LibVLCSharp.Forms/Shared/Band.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace LibVLCSharp.Forms.Shared
```

Comment:
If this does not depend on XF and it may be useful to other GUI integrations, it should be located in the main LibVLCSharp package, likely at `libvlcsharp/src/LibVLCSharp/Shared/MediaPlayerElement`

Reply:
Should I keep the INPC inheritance ?

Reply:
I guess not, this smells like a `Model` class, doesn't it?

Reply:
I use these classes as `ViewModel` .